### PR TITLE
Add framerate param

### PR DESCRIPTION
**Pull request by ArjunPutcha**
Add framerate param to cnmf and cnmfe params gui

### CODE REVIEW

File: mesmerize_napari/cnmf_gui.py [hunk 1]
```diff
@@ -38,7 +38,7 @@ def get_params(self, *args, group_params: bool = True) -> Tuple[str, bool, dict]
             "tsub": self.ui.spinBox_tsub.value(),
             "method_init": self.ui.comboBox_method_init.currentText(),
             # 'border_pix': bord_px,
-            # 'fr': self.vi.viewer.workEnv.imgdata.meta['fps']
+            'fr': self.ui.doubleSpinBoxFrameRate.value(),
```
Comment (kushalkolar) [inline mesmerize_napari/cnmf_gui.py:41]: at the beginning of `get_params()` I would do a check for the framerate value and raise if it is `<= 0`

File: mesmerize_napari/cnmfe_gui.py [hunk 2]
```diff
@@ -62,6 +62,7 @@ def _add_item_cnmfe(self, *args, group_params: bool = True):
             "method_deconvolution": method_deconvolution,
             "update_background_components": True,
             "del_duplicates": True,
+            "fr": self.ui.doubleSpinBoxFrameRate.value(),
```
Comment (kushalkolar) [inline mesmerize_napari/cnmfe_gui.py:65]: similar to CNMF, put a check at the top of the `get_params()` to raise if `<= 0`

File: mesmerize_napari/main_offline_gui.py [hunk 3]
```diff
@@ -411,20 +411,35 @@ def view_downsample_mcorr(self):
     def view_shifts(self):
         s = self.selected_series()
         if s["params"]["mcorr_kwargs"]["pw_rigid"]:
-            x_shifts, y_shifts = s.mcorr.get_shifts(output_type="napari-1d")
-            self.viewer.add_image(x_shifts, name=f'{s["name"]}: X shifts')
-            self.viewer.add_image(y_shifts, name=f'{s["name"]}: Y shifts')
+            shifts = s.mcorr.get_shifts()
```
Comment (kushalkolar) [inline mesmerize_napari/main_offline_gui.py:414]: you should've checked out a new branch from master for this :P

File: mesmerize_napari/napari1d_manager.py [hunk 4]
```diff
@@ -247,8 +247,9 @@ def __init__(self, batch_item: pd.Series):
             self.plot_els_shifts()
 
     def plot_rig_shifts(self):
-        xs, ys = self.batch_item.mcorr.get_shifts(
-            output_type="napari-1d", pw_rigid=False
+        shifts = self.batch_item.mcorr.get_shifts()
```
Comment (kushalkolar) [inline mesmerize_napari/napari1d_manager.py:250]: should've checked out a new branch from master for this :P

**Review by kushalkolar — CHANGES_REQUESTED** [review]
1. I would set the default value for the framerate in the GUI as -1 to force the user to set it.
2. I'm assuming tab order has been checked?

File: mesmerize_napari/main_offline_gui.py (hunk 3 above)
Comment (ArjunPutcha) [reply mesmerize_napari/main_offline_gui.py:414]: Ah shit I thought I did, my bad!

File: mesmerize_napari/cnmfe_gui.py (hunk 2 above)
Comment (ArjunPutcha) [reply mesmerize_napari/cnmfe_gui.py:65]: good idea

**Comment by kushalkolar** [comment]
@ArjunPutcha since this PR contains commits from another branch, checkout a new branch from master, copy over the appropriate changes specific to this PR, and make a new PR from that new branch.

**Pull request closed without merging**